src/commons/services/utils.js

```diff
@@ -88,6 +90,25 @@
             });
         };
 
+        vm.getClusterDetails = function(clusterId) {
+            var cluster = {"name" : "Unassigned"};
+            if(vm.clusterData !== null && clusterId !== "") {
+                var i, key, clusterData, len;
+                clusterData = vm.clusterData.clusters;
+                len = clusterData.length;
+                for ( i = 0; i < len; i++ ) {
+                    var clusterObj = clusterData[i];
+                    for( key in clusterObj ) {
+                        if(key !== "stats" && clusterObj[key].tendrl_context.cluster_id == clusterId) {
```

Comment:
Use triple equal to for "clusterObj[key].tendrl_context.cluster_id == clusterId"